tools/swift-inspect/Sources/swift-inspect/Operations/DumpConcurrency.swift

```diff
@@ -313,19 +313,19 @@ fileprivate class ConcurrencyDumper {
     let taskToThread: [swift_addr_t: UInt64] =
         Dictionary(threadCurrentTasks.map{ ($1, $0) }, uniquingKeysWith: { $1 })
 
-    var lastChilds: [Bool] = []
+    var lastChildren: [Bool] = []
```

Comment:
While grammatically incorrect, I think that in general, this type of change is less desirable as it is changing terminology for variables rather than correcting an obvious typo. In this specific case, the rename is fine, but it is not always so obvious.

Reply:
+1. This array _isn't_ an array of children. It's an array of "last child" flags. `lastChildren` is wrong. A better name might be `lastChildFlags`.

Reply:
Fixed!